vllm/core/scheduler.py

```diff
@@ -286,6 +295,13 @@ def __init__(
         # Latency of the last prompt step
         self.last_prompt_latency = 0.0
 
+        # The following field is test-only. It is used to inject artificial
+        # preemption.
+        self.enable_artificial_preemption = ENABLE_ARTIFICIAL_PREEMPT
```

Comment:
could be premature optimization, but usuallly accessing global var is more expensive than class attribute, so I just moved it here